tests/func/test_install.py

```diff
@@ -30,6 +30,13 @@ def test_should_create_hooks(self, git, dvc_repo):
             with open(self._hook(fname), "r") as fobj:
                 assert command in fobj.read()
 
+    def test_should_fail_if_file_already_exists(self, git, dvc_repo):
+        with open(self._hook("post-checkout"), "w") as fobj:
+            fobj.write("hook content")
+
+        assert main(["install"]) != 0
+
+    @pytest.mark.skip(reason="https://github.com/iterative/dvc/issues/2362")
```

Comment:
Leftover from WIP? These tests should be removed now, right?

Reply:
Well, to be honest looking at #2362 I though that this change is first step to handle hooks more gracefully. I though that we might want to get back to this tests at some point, and thats why I thought leaving them around might be a good think. What do you think?

Reply:
@pared No need to keep unused code, we can bring this back later if we need to without any problems.